tests/unit/test_match_users.py

```diff
@@ -127,11 +184,11 @@ def test_multiple_match(self):
 
         # Check for correct response
         res_json = response.get_json()
-        expected_res = [self.journey_one, self.journey_two]
-        res_json = sorted(res_json, key=lambda d: d['UserId'])
 
-        for i in range(len(res_json)):
-            assert sorted(res_json[i].items()) == sorted(expected_res[i].items())
+        for maps in res_json:
+            currUserID = maps["id"]
+            for key, value in maps.items():
```

Comment:
Same here!

Reply:
Can do the same thing here but it's not as clean or easy to change if you want to test for more than 2 matches (three total users). I suggest renaming the function to test_two_matches and adding in an assertion for len again. `assert len(res_json)==2`

With these changes the tests fail, if you add in journey 4 for example. The issue was that the the dict was checking for empty mappings which where giving equivalences when checked against empty mappings.

Reply:
`test_multiple_match` should ideally be a function that checks for a random number of matches between 1-5 for example. Can do this by creating a dictionary/map for the journeys as well and calling for it based on the random number generated.